common/src/main/scala/org/apache/celeborn/common/RssConf.scala

```diff
@@ -713,17 +903,92 @@ object RssConf extends Logging {
+    conf.get(HA_MASTER_RATIS_LOG_APPENDER_QUEUE_NUM_ELEMENTS)
+
+  def haMasterRatisLogAppenderQueueBytesLimit(conf: RssConf): Long =
+    conf.get(HA_MASTER_RATIS_LOG_APPENDER_QUEUE_BYTE_LIMIT)
+
+  def haMasterRatisLogPurgeGap(conf: RssConf): Int = conf.get(HA_MASTER_RATIS_LOG_PURGE_GAP)
+
+  def haMasterRatisRpcRequestTimeout(conf: RssConf): Long =
+    conf.get(HA_MASTER_RATIS_RPC_REQUEST_TIMEOUT)
+
+  def haMasterRatisRetryCacheExpiryTime(conf: RssConf): Long =
+    conf.get(HA_MASTER_RATIS_SERVER_RETRY_CACHE_EXPIRY_TIME)
+
+  def haMasterRatisRpcTimeoutMin(conf: RssConf): Long = conf.get(HA_MASTER_RATIS_RPC_TIMEOUT_MIN)
+
+  def haMasterRatisRpcTimeoutMax(conf: RssConf): Long = conf.get(HA_MASTER_RATIS_RPC_TIMEOUT_MAX)
+
+  def haMasterRatisNotificationNoLeaderTimeout(conf: RssConf): Long =
+    conf.get(HA_MASTER_RATIS_NOTIFICATION_NO_LEADER_TIMEOUT)
+
+  def haMasterRatisRpcSlownessTimeout(conf: RssConf): Long =
+    conf.get(HA_MASTER_RATIS_RPC_SLOWNESS_TIMEOUT)
+
+  def haMasterRatisRoleCheckInterval(conf: RssConf): Long =
+    conf.get(HA_MASTER_RATIS_ROLE_CHECK_INTERVAL)
+
+  def haMasterRatisSnapshotAutoTriggerEnabled(conf: RssConf): Boolean =
+    conf.get(HA_MASTER_RATIS_SNAPSHOT_AUTO_TRIGGER_ENABLED)
+
+  def haMasterRatisSnapshotAutoTriggerThreshold(conf: RssConf): Long =
+    conf.get(HA_MASTER_RATIS_SNAPSHOT_AUTO_TRIGGER_THRESHOLD)
+  def haMasterRatisSnapshotRetentionFileNum(conf: RssConf): Int =
+    conf.get(HA_MASTER_RATIS_SNAPSHOT_RETENTION_FILE_NUM)
+
+  val WORKER_REPLICATE_THREADS: ConfigEntry[Int] =
+    buildConf("celeborn.worker.replicate.threads")
+      .withAlternative("rss.worker.replicate.numThreads")
+      .categories("worker")
+      .doc("Thread number of worker to replicate shuffle data.")
+      .intConf
+      .createWithDefault(64)
+
+  val WORKER_COMMIT_THREADS: ConfigEntry[Int] =
+    buildConf("celeborn.worker.commit.threads")
+      .withAlternative("rss.worker.asyncCommitFiles.numThreads")
+      .categories("worker")
+      .doc("Thread number of worker to commit shuffle data files asynchronously.")
+      .intConf
+      .createWithDefault(32)
+
+  val WORKER_STORAGE_DIRS: OptionalConfigEntry[Seq[String]] =
+    buildConf("celeborn.worker.storage.dirs")
+      .withAlternative("rss.worker.base.dirs")
+      .categories("worker")
+      .doc("Directory list to store shuffle data. Storage size limit can be set for each " +
+        "directory. For the sake of performance, there should be no more than 2 directories " +
+        "on the same disk partition if you are using HDD. There can be 4 or more directories " +
+        "can run on the same disk partition if you are using SSD. For example: " +
+        "dir1[:capacity=][:disktype=][:flushthread=],dir2[:capacity=][:disktype=][:flushthread=]")
+      .stringConf
+      .toSequence
+      .createOptional
+
+  val WORKER_FLUSH_BUFFER_SIZE: ConfigEntry[Long] =
+    buildConf("celeborn.worker.flush.buffer.size")
```

Comment:
I see other places using `rss.fluster.*`, maybe we should unify them into `celeborn.worker.flush.*` later